blocks/bricks/sequence_generators.py

```diff
@@ -270,8 +272,18 @@ def cost_matrix(self, application_call, outputs, mask=None, **kwargs):
         feedback = tensor.set_subtensor(
             feedback[0],
             self.readout.feedback(self.readout.initial_outputs(batch_size)))
+
+        # Run the language model
+        if self.language_model:
+            lm_arguments = self.language_model.evaluate(
+                outputs=outputs, mask=mask, as_dict=True)
+            lm_arguments = dict_subset(lm_arguments, ['outputs'])
```

Comment:
I think that a shift similar to the one in the line 267 should be applied to what currently is called `lm_arguments`. Do  you agree?